.github/workflows/lagom-template.yml

```diff
@@ -113,6 +113,12 @@ jobs:
           download_cache_path: ${{ github.workspace }}/Build/caches
           vcpkg_cache_path: ${{ github.workspace }}/Build/caches/vcpkg-binary-cache
 
+      - name: Enable read-write vcpkg source assets cache
+        if: github.ref == 'refs/heads/master'
+        run: |
+          echo "VCPKG_CACHE_SAS=${{ secrets.VCPKG_CACHE_SAS }}" >> "$GITHUB_ENV"
```

Comment:
I think conventional wisdom is to export secrets as close to time of use as possible. ... This is pretty close though. 

This step will export the secret into the environment for all subsequent steps, but we only "need" it in the env for the build steps. 

So perhaps putting it in the env: dictionary for the individual build steps (all half dozen of them for some reason) would be 'safer'?

Reply:
Hm, I did this since I only want to set those env vars for `master`. Can we do that in `env:` as well? 🤔

Reply:
Uh. That's a good question. I don't think env: works that way

Reply:
Updated - there were only two steps, that's still manageable I'd say. As a bonus, I also found a way to unify our different build steps into one.